packages/fxa-content-server/app/scripts/templates/settings/delete_account.mustache

```diff
@@ -11,7 +14,50 @@
     <div class="error"></div>
 
     <form novalidate>
-      <p>{{#t}}If you use this account for Mozilla apps and services like AMO, Pocket, and Screenshots you should first log in and remove any saved personal information. Once you delete your account here you may lose access to it everywhere.{{/t}}</p>
+      <p>{{#t}}Your Firefox Account gives you access to products and services that help keep your web experience secure, private, and productive.{{/t}}</p>
+
+      <p>{{#t}}You currently use:{{/t}}</p>
+
+        <ul class="account-summary-list">
+          {{#subscriptions}}
+            <li>
+              {{productName}}
```

Comment:
Oof, [I wonder if this bit of tech debt is already starting to bite us](https://github.com/mozilla/fxa/issues/1187)? 😓  Despite the name "name", this is actually an ID and not a human-readable description.

Now that I belatedly think of it, the resource that `account.fetchActiveSubscriptions()` requests doesn't actually include human-readable descriptions. That particular resource is used [to build subscription capability lists for profile data](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/utils/subscriptions.js#L25).

On the payments-server pages, we use [the customer data from subhub](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/subscriptions.js#L209) which lists subscriptions that [*do* have human readable names](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/validators.js#L303). We might need a revisit of PR #1715 and PR #1773 to `GET /oauth/subscriptions/customer`

Reply:
There is overlap here with what I'm doing for #970 too, I now realise. I'm adding a new endpoint that returns, among other things, the result from `subhub.listSubscriptions` (so that we can check for `failure_code` properties and display an error message). Seems like the same data would be useful here (for different reasons).

Reply:
Oh maybe not, `listSubscriptions` returns `plan_name` but not `product_name`. I guess that's different?

Reply:
Ah, yeah, I definitely assumed that was the human-readable _name_. 😅 It sounds like we might need to address #1187 + add a new field if what Phil is working on/`plan_name` is not what we need?

Reply:
> It sounds like we might need to address #1187 + add a new field

Addressing #1187 won't really help here much. And I don't think we *want* to add a human-readable label to the FxA database: we'd only ever use it where subhub APIs are also available - and they already have the labels.

> Oh maybe not, listSubscriptions returns plan_name but not product_name. I guess that's different?

Using `listSubscriptions` is probably what we want. The `plan_name` describes the user's actual subscription plan - e.g. "123Done Pro Plus Monthly". A `product_name` would describe what the plan grants - e.g. "123Done with custom emoji enabled".

If we really, really need `product_name` we would need to either a) request that subhub make a change to include `product_name` or b) do what we do in payments-server by also fetching `listPlans` and looking up the `product_name` for the respective `plan_name` there.

Reply:
Then perhaps I just need to check with UX that displaying `plan_name` is along the line of what they were envisioning... They seem flexible with this section.

@philbooth, is that something you're working on now?

Reply:
Yeah, let split it out to a separate PR to get it through review quicker...

Reply:
Sorry for the delay, took me longer to write some passing tests than I expected (always the way). Here you go:

https://github.com/mozilla/fxa/pull/1824